dev/reference/best-practices/high-concurrency.md

```diff
@@ -0,0 +1,211 @@
+
+Given this feature and the need to avoid the hotspot problem, TiDB offers the `Split Region` syntaxes for optimizing the data write in the high concurrent and write intensive scenario. Based on the above case, now you can try to scatter Regions using the `Split Region` syntaxes and observe the load distribution.
+
+Because the data to be written in the test is entirely discrete within the positive range, you can use the following statement to pre-split the table into 128 Regions within the range of `minInt64` and `maxInt64`:
+
+{{< copyable "sql" >}}
+
+```sql
+SPLIT TABLE TEST_HOTSPOT BETWEEN (0) AND (9223372036854775807) REGIONS 128;
+```
+
+After the pre-split operation, execute the `SHOW TABLE test_hotspot REGIONS;` statement to check the status of Region scattering. If the values of the `SCATTERING` column are all `0`, the scheduling is successful.
+
+You can also check the Region distribution using the [table-regions.py](https://github.com/pingcap/tidb-ansible/blob/dabf60baba5e740a4bee9faf95e77563d8084be1/scripts/table-regions.py) script. Currently, the Region distribution is relatively even:
+
+```
+[root@172.16.4.4 scripts]# python table-regions.py --host 172.16.4.3 --port 31453 test test_hotspot
+[RECORD - test.test_hotspot] - Leaders Distribution:
+  total leader count: 127
+  store: 1, num_leaders: 21, percentage: 16.54%
+  store: 4, num_leaders: 20, percentage: 15.75%
+  store: 6, num_leaders: 21, percentage: 16.54%
+  store: 46, num_leaders: 21, percentage: 16.54%
+  store: 82, num_leaders: 23, percentage: 18.11%
+  store: 62, num_leaders: 21, percentage: 16.54%
+```
+
+Then operate the write load again:
+
+![QPS6](/media/best-practices/QPS6.png)
+
+![QPS7](/media/best-practices/QPS7.png)
+
+![QPS8](/media/best-practices/QPS8.png)
+
+You can see that the hotspot has been eliminated now.
+
+In this case, the table is simple. In other cases, you might also need to consider the hotspot problem of index. For more details on how to pre-split the index Region, refer to [Split Region](/dev/reference/sql/statements/split-region.md).
+
+## More complex hotspot problem
+
+If a table does not have a primary key, or the primary key is not the `Int` type and you do not want to generate a randomly distributed primary key ID, TiDB provides an implicit `_tidb_rowid` column as the row ID. If you do not use the `SHARD_ROW_ID_BITS` parameter, the values of the `_tidb_rowid` column might be monotonically increasing, which also causes hotspot. Refer to [`SHARD_ROW_ID_BITS` description](/dev/reference/configuration/tidb-server/tidb-specific-variables.md#shard_row_id_bits) for more detail.
+
+To avoid the hotspot problem in this situation, you can use `SHARD_ROW_ID_BITS` and `PRE_SPLIT_REGIONS` when creating a table. For more details about `PRE_SPLIT_REGIONS`, refer to [here](/dev/reference/sql/statements/split-region.md#pre_split_regions).
```

Comment:
Avoid `here`. Use the specific description instead.

```suggestion
To avoid the hotspot problem in this situation, use `SHARD_ROW_ID_BITS` and `PRE_SPLIT_REGIONS` when creating a table. For more details about `PRE_SPLIT_REGIONS`, refer to [Pre-split regions](/dev/reference/sql/statements/split-region.md#pre_split_regions).
```